### DISCUSSION TITLE
embed url into title

### DISCUSSION THREAD

**Discussion by chrispalka** [JoeBanks13/webhook-discord#67] [Q&A] [answered]
Not really an issue, but was wondering if there was a way to embed url into a title.

Similar to how discord api documents it below:

{
  "embeds": [{
    "title": "Google it!",
    "url": "https://google.com/"
  }]
}

**Comment by jb3** [comment, marked as answer]
Yep! Check out the [`setURL`](https://jb3.github.io/webhook-discord/MessageBuilder.html#setURL) method of the message builder.

Please feel free to re-open if you have any further questions.

**Reply by chrispalka** [reply]
Thank you! I was trying that, however I must be doing something wrong.

What I have is below:

`.addField(title, '', false)
    .setURL(productLink)`

Essentially I want the title within addField linked to a specific URL.

**Reply by jb3** [reply]
Ah, I'm afraid that's a limitation at Discord's end, you can't hyperlink field titles.

What you can do is add links into the body using markdown like `[example hyperlink text](https://example.com/)`.

The example you reference in the above issue is to set the overall embed title, you can't set individual field titles.
